src/app/applications/applications.component.html

```diff
@@ -1129,7 +1132,9 @@ <h6 class="col-md-8 form-control-label">
 									<div
 										*ngIf="
 											fl.type.shortcut === t.shortcut &&
-											(fl.simple_vm || selectedApplication?.project_application_openstack_project)
+											(fl.simple_vm ||
+												selectedApplication?.project_application_openstack_project ||
+												fl.type.shortcut === 'cstm')
```

Comment:
cstm  should not be hardcoded in html (we should use a variable or like in the new platform an enum (https://github.com/deNBI/simplevm-webapp-draft/blob/master/src/app/shared/enums/flavor_type_shortcuts.ts)

Which mirrors the API entries